lib/ocso_scraper/page_parser.rb

```diff
@@ -0,0 +1,38 @@
+require 'nokogiri'
+module OcsoScraper
+  class PageParser
```

Comment:
this file seems to be my based on my code from my crawler_parser but implemented in the scraper gem, with a parse date and strip function correct?

Reply:
yup!